test/support/file_endpoint.ex

```diff
@@ -65,12 +55,13 @@ defmodule Membrane.RTC.Engine.Support.FileEndpoint do
         source: %Membrane.File.Source{
           location: state.file_path
         },
-        realtimer: Membrane.Realtimer
```

Comment:
Are you sure about removing realtimer? I think that without it `FileEndpoint` can lead to `Toilet overflow` in receiving endpoints.

Reply:
I didn't observer any problems :thinking: It works fine on CI

Reply:
As we discussed this test will be fixed in another PR